src/arch/xtensa/configs/override/tigerlake_chrome_rtnr.config

```diff
@@ -0,0 +1,9 @@
+CONFIG_TIGERLAKE=y
+CONFIG_INTEL_DMIC=y
+CONFIG_INTEL_SSP=y
+CONFIG_INTEL_ALH=y
+CONFIG_LP_MEMORY_BANKS=1
+CONFIG_HP_MEMORY_BANKS=46
```

Comment:
configs in line#1~6 don't need in override config file. They are already in tgplp_defconfig.
You could refer to tigerlake_chrome_igonr.config.

Reply:
Duplicated configs are removed. Thanks.